*** auto-tests/A-18-13-M-ok.out	2020-11-26 14:09:33.360000000 +0000
--- auto-tests/A-18-13-M-ok.outhyp	2020-12-08 02:03:48.929798822 +0000
***************
*** 59,65 ****
  4 - Activa/desactivar notificações de um produto
  5 - Mostrar transacções do cliente
  0 - Sair
! Escolha uma opção: Identificador do cliente: R2|Jorge Figueiredo|Lisboa, Portugal|ELITE|3000|2820
  Gestão de Clientes
  1 - Mostrar cliente
  2 - Mostrar clientes
--- 59,65 ----
  4 - Activa/desactivar notificações de um produto
  5 - Mostrar transacções do cliente
  0 - Sair
! Escolha uma opção: Identificador do cliente: R2|Jorge Figueiredo|Lisboa, Portugal|ELITE|3000|2940
  Gestão de Clientes
  1 - Mostrar cliente
  2 - Mostrar clientes
***************
*** 85,91 ****
  3 - Registar encomenda
  4 - Pagar
  0 - Sair
! Escolha uma opção: Identificador da transacção: 2|R2|B|6|2400|2280|2|0
  Gestão de Transacções
  1 - Visualizar
  2 - Registar venda
--- 85,91 ----
  3 - Registar encomenda
  4 - Pagar
  0 - Sair
! Escolha uma opção: Identificador da transacção: 2|R2|B|6|2400|2160|2|0
  Gestão de Transacções
  1 - Visualizar
  2 - Registar venda
